db/re/item_cardalbum.txt

```diff
@@ -374,73 +379,116 @@ IG_CardAlbum,4448,1		// Cornus Card
+IG_CardAlbum,4569,1		// Chen Liu Card
+IG_CardAlbum,4570,1		// Flamel Emure Card
+IG_CardAlbum,4571,1		// Gertie Wie Card
+IG_CardAlbum,4572,1		// Randel Lawrence Card
+IG_CardAlbum,4573,1		// Trentini Card
+IG_CardAlbum,4575,1		// Soheon Card
+IG_CardAlbum,4577,1		// Elvira Card
+IG_CardAlbum,4579,1		// Lora Card
+IG_CardAlbum,4581,1		// Rudo Card
+IG_CardAlbum,4582,1		// Bungisngis Card
+IG_CardAlbum,4583,1		// Engkanto Card
+IG_CardAlbum,4584,1		// Manananggal Card
+IG_CardAlbum,4585,1		// Mangkukulam Card
+IG_CardAlbum,4586,1		// Tikbalang Card
+IG_CardAlbum,4587,1		// Tiyanak Card
+IG_CardAlbum,4588,1		// Wakwak Card
+IG_CardAlbum,4589,1		// Jejeling Card
+IG_CardAlbum,4593,1		// Menblatt Card
+IG_CardAlbum,4594,1		// Petal Card
+IG_CardAlbum,4595,1		// Cenere Card
+IG_CardAlbum,4596,1		// Antique Book Card
+IG_CardAlbum,4597,1		// Lichtern Blue Card
+IG_CardAlbum,4598,1		// Lichtern Yellow Card
+IG_CardAlbum,4599,1		// Lichtern Red Card
+IG_CardAlbum,4600,1		// Lichtern Green Card
+IG_CardAlbum,4626,1		// Big Ben Card
+IG_CardAlbum,4627,1		// Big Bell Card
+IG_CardAlbum,4628,1		// Neo Punk Card
+IG_CardAlbum,4629,1		// Arc Elder Card
+IG_CardAlbum,4630,1		// Timer Keeper Card
+IG_CardAlbum,4631,1		// Owl Viscount Card
+IG_CardAlbum,4632,1		// Owl Marquees Card
+
 // Magic Card Album
-IG_MagicCardAlbum,4047,1	// Ghostring Card
-IG_MagicCardAlbum,4054,1	// Angeling Card
-IG_MagicCardAlbum,4163,1	// Gryphon Card
-IG_MagicCardAlbum,4169,1	// Dark Illusion Card
-IG_MagicCardAlbum,4174,1	// Deviling Card
-IG_MagicCardAlbum,4179,10	// Dragon Fly Card
-IG_MagicCardAlbum,4183,10	// Vagabond Wolf Card
-IG_MagicCardAlbum,4197,10	// Mastering Card
-IG_MagicCardAlbum,4198,1	// Maya Purple Card
-IG_MagicCardAlbum,4203,10	// Mutant Dragonoid Card
-IG_MagicCardAlbum,4207,1	// Mysteltainn Card
-IG_MagicCardAlbum,4211,1	// Vocal card
-IG_MagicCardAlbum,4237,5	// Owl Duke Card
-IG_MagicCardAlbum,4238,5	// Owl Baron Card
-IG_MagicCardAlbum,4241,1	// Arc Angeling Card
-IG_MagicCardAlbum,4250,1	// Executioner Card
-IG_MagicCardAlbum,4254,1	// Tirfing Card
-IG_MagicCardAlbum,4266,10	// Eclipse Card
-IG_MagicCardAlbum,4284,10	// Chepet Card
-IG_MagicCardAlbum,4285,10	// Choco Card
-IG_MagicCardAlbum,4290,5	// Cat O' Nine Tails Card
-IG_MagicCardAlbum,4291,10	// Kobold Leader Card
-IG_MagicCardAlbum,4300,10	// Chimera Card
-IG_MagicCardAlbum,4306,10	// Toad Card
-IG_MagicCardAlbum,4320,1	// Bloody Knight Card
-IG_MagicCardAlbum,4336,1	// Ungoliant Card
-IG_MagicCardAlbum,4354,1	// Gemini-S58 Card
-IG_MagicCardAlbum,4384,1	// Hydro Card
-IG_MagicCardAlbum,4395,3	// Thanatos Maero Card
-IG_MagicCardAlbum,4396,3	// Thanatos Odium Card
-IG_MagicCardAlbum,4397,3	// Thanatos Despero Card
+IG_MagicCardAlbum,4047,3	// Ghostring Card
+IG_MagicCardAlbum,4054,3	// Angeling Card
+IG_MagicCardAlbum,4163,14	// Gryphon Card
+IG_MagicCardAlbum,4169,14	// Dark Illusion Card
+IG_MagicCardAlbum,4174,3	// Deviling Card
+IG_MagicCardAlbum,4179,140	// Dragon Fly Card
```

Comment:
Shouldn't the rates be 10?

Reply:
Ah one thing I forgot to mention was that the rates for `IG_MagicCardAlbum` were based on the Natural Random values listed on DP: 
![image](https://user-images.githubusercontent.com/7020503/101532682-004ad700-39e9-11eb-968f-f8394c022f6e.png)

@Atemo Should I be referring to the regular random values instead?
![image](https://user-images.githubusercontent.com/7020503/101532761-1b1d4b80-39e9-11eb-9977-e6b227095926.png)

Reply:
Yes, the value on rathena should match the normal values